marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow.sjs

```diff
@@ -143,8 +135,8 @@ class Flow {
   findMatchingContent(flowName, stepNumber, options, filterQuery) {
     const flow = this.getFlow(flowName);
     const flowStep = flow.steps[stepNumber];
-    const stepDetails = this.step.getStepByNameAndType(flowStep.stepDefinitionName, flowStep.stepDefinitionType);
-    const combinedOptions = Object.assign({}, stepDetails.options || {}, flow.options || {}, flowStep.options || {}, options);
+    const stepDefinition = this.step.getStepByNameAndType(flowStep.stepDefinitionName, flowStep.stepDefinitionType);
```

Comment:
I opted for "stepDefinition" and "flowStep" to distinguish between step config in the step definition document, and step config in the step as it's defined in a flow. "stepDetails" confused me too much.